post_office/admin.py

```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
please remove the two lines above. In python-3 they don't make any sense.

Reply:
I'm not sure where those are coming from to be honest. I've never added them so it might be related to git merging things